site/content/docs/main/restore-reference.md

```diff
@@ -3,27 +3,13 @@ title: "Restore Reference"
 layout: docs
 ---
 
-## Restoring Into a Different Namespace
-
-Velero can restore resources into a different namespace than the one they were backed up from. To do this, use the `--namespace-mappings` flag:
-
-```bash
-velero restore create RESTORE_NAME \
-  --from-backup BACKUP_NAME \
-  --namespace-mappings old-ns-1:new-ns-1,old-ns-2:new-ns-2
-```
-## What happens when user removes restore objects
-A **restore** object represents the restore operation. There are two types of deletion for restore objects:
-1. Deleting with **`velero restore delete`**.
-This command will delete the custom resource representing it, along with its individual log and results files. But, it will not delete any objects that were created by it from your cluster.
-2. Deleting with **`kubectl -n velero delete restore`**.
-This command will delete the custom resource representing the restore, but will not delete log/results files from object storage, or any objects that were created during the restore in your cluster.
+The page outlines how to use the `velero restore` command, configuration options for restores, and describes the main process Velero uses to perform restores.
 
 ## Restore command-line options
-To see all commands for restores, run : `velero restore --help`
-To see all options associated with a specific command, provide the --help flag to that command. For example,  **`velero restore create --help`** shows all options associated with the **create** command.
+To see all commands for restores, run `velero restore --help`
+To see all options associated with a specific command, provide the --help flag to that command. For example,  `velero restore create --help` shows all options associated with the `create` command.
 
-To list all options of restore, use **`velero restore --help`**
+To list all options of restore, use `velero restore --help`.
```

Comment:
This is redundant with the sentence that starts this paragraph "To see all commands for restores...."